README.md

```diff
@@ -10,35 +10,87 @@ Compile, edit namelist, and run canopy model:
 
 Current Canopy-App components:
 
-1.  In-Canopy Winds and Wind Adjustment Factor (WAF) for wildland fire spread model applications (Massman et al., 2017).
+1.  In-Canopy Winds and Wind Adjustment Factor (WAF) for wildland fire spread and air quality applications.
+
+    Namelist Option : `ifcanwind`
 
     - `canopy_wind_mod.F90`
     - `canopy_waf_mod.F90`
 
-2.  In-Canopy radiative transfer (i.e., photolysis rate adjusment) for air quality applications (in progress...)  (Makar et al., 2017)
-
-    - `canopy_prad_mod.F90`
-
-3.  In-Canopy vertical diffusion (i.e., eddy diffusivity adjustment) for air quality applications (in progress...) (Makar et al., 2017)
-
-    - `canopy_vdiff_mod.F90`
-
-
-Citations:
+2.  In-Canopy vertical diffusion (i.e., eddy diffusivity adjustment) for air quality applications.
+
+    Namelist Option : `ifcaneddy`
+
+    - `canopy_eddyx_mod.F90`
+
+3.  In-Canopy vertical photolysis attenuation for air quality applications.
+
+    Namelist Option : `ifcanphot`
+
+    - `canopy_phot_mod.F90`
+
+Current Canopy-App inputs:
+
+    Typical 2D gridded atmospheric model input variables in 1st layer above canopy (e.g., `input_variables.txt`).
+
+    Namelist Option : `file_vars`
+
+    | Variable Name    | Variable Descrip. (Units)                         |
+    | ---------------  | ------------------------------------------------- |  
+    | LAT              | Latitude  (degrees)                               |
+    | LON              | Longitude (degrees)                               |
+    | FH               | Forest canopy height (m)                          |
+    | WS               | Wind speed at reference height (m/s), e.g., 10 m  |
+    | CLU              | Forest clumping index (dimensionless)             |
+    | LAI              | Leaf area index (m2/m2)                           |
+    | VTYPE            | Vegetation type (dimensionless), e.g., VIIRS      |
+    | FFRAC            | Forest fraction (dimensionless)                   |
+    | UST              | Friction velocity (m/s)                           |
+    | CSZ              | Cosine of the solar zenith angle (dimensionless)  |
+    | Z0               | Total surface roughness length (m)                |
+    | MOL              | Monin-Obukhov Length (m)                          |
+    | FRP              | Total Fire Radiative Power (MW/grid cell)         |
+
+    Current User Namelist Options:
+
+    | Variable Name    | Variable Descrip. (Units)                                                          |
+    | ---------------  | ---------------------------------------------------------------------------------- |
+    | nlat             | number of latitude cells (must match # of LAT in `file_vars` above )               |
+    | nlon             | number of longitude cells (must match # of LON in `file_vars`above )               |
+    | canlays          | number of canopy layers                                                            |
```

Comment:
doesn't the vertical grid include layers above the canopy as well? maybe `canlays` isn't the best name in that case

Reply:
I completely agree with you, and have changed the variable name to "modlays" if you agree in the next [commit](https://github.com/noaa-oar-arl/canopy-app/pull/20/commits/508b5f2f5008f59a7a2ef5298ee68c79e7c5a4ff).

Reply:
@zmoon Is this OK to resolve?  Note.  I have also changed "canres" to "modres".